src/ws/pam_cockpit_cert.c

```diff
@@ -22,179 +22,15 @@
 
 #include "config.h"
 
-#include <assert.h>
```

Comment:
Can you please add a comment to the file that it is a stub for old/modified /etc/pam.d/cockpit files, is deprecated, and will be removed eventually? Just to unconfuse our future selves.